.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "rust-analyzer.linkedProjects": ["bindings_ffi/Cargo.toml"]
```

Comment:
🤯 

Didn't know this was an option. So, we can have separate builds but still have VSCode run Rust Analyzer on all the projects